commons-packet/commons-packet-service/src/main/java/io/mosip/commons/packetmanager/service/PacketReaderService.java

```diff
@@ -162,16 +162,20 @@ else if (e instanceof BaseCheckedException) {
         }
     }
 
-    private String getKey() throws IOException {
-        if (key != null)
-            return key;
-        JSONObject jsonObject = getMappingJsonFile();
-        if(jsonObject == null)
-            return null;
-        LinkedHashMap<String, String> individualBio = (LinkedHashMap) jsonObject.get(INDIVIDUAL_BIOMETRICS);
-        key = individualBio.get(VALUE);
-        return key;
-    }
+	private String getKey() throws IOException {
+		if (key != null)
+			return key;
+		JSONObject jsonObject = getMappingJsonFile();
+		if (jsonObject == null)
```

Comment:
Remove the null check here. The sonar issue refers that the null check is not required as it always holds some object and will never be null.